sequencing/vcf_slice/dxapp.json

```diff
@@ -1,69 +1,90 @@
 {
   "name": "vcf_slice",
   "title": "Slice VCF File(s)",
-  "summary": "vcf_slice",
+  "summary": "Return a small section of a VCF file or files (similar to tabix).",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
       "name": "vcf_fn",
-      "label": "VCF Files",
+      "label": "VCF files",
+      "help": "Variant files to be sliced, in VCF format.",
       "class": "array:file",
       "optional": false,
-      "patterns" : ["*.vcf.gz"]
+      "patterns": ["*.vcf.gz"]
     },
     {
       "name": "vcfidx_fn",
-      "label": "VCF Index Files",
+      "label": "VCF index files",
+      "help": "Tabix index files corresponding to VCF files. Each VCF must have a corresponding index with matching base filename.",
       "class": "array:file",
       "optional": false,
-      "patterns" : ["*.vcf.gz.tbi"]
+      "patterns": ["*.vcf.gz.tbi"]
     },
     {
       "name": "region_fn",
       "label": "File of regions (chr:start-stop)",
+      "help": "Text file of genomic region coordinates. Variants within these ranges will be extracted from the given VCF files.",
       "class": "file",
-      "optional": false
+      "optional": false,
+      "patterns": ["*.txt", "*.regions", "*.list"]
     },
     {
-      "name": "suffix_fn",
-      "label": "Output File name suffix (Optional)",
+      "name": "suffix",
+      "label": "Output file name suffix (optional)",
+      "help": "(Optional) Include this given string in the output VCF filename extension, e.g. '*.sliced.SUFFIX.vcf.gz'.",
       "class": "string",
       "optional": true
     }
   ],
   "outputSpec": [
     {
       "name": "vcf_out",
```

Comment:
"vcf_out" -> "out_variants_vcfgzs"
The variable name seems to be misplaced with "out_variants_vcfgztbis" (below)

Also need to update variables accordingly in the app script.